### DISCUSSION TITLE
Unable to connect/sync with atlassian jira cloud version

### DISCUSSION THREAD

**Discussion by Pillmayr** [super-productivity/super-productivity#2520] [Q&A]
Hello,
I really like this tool (Version 7.11.5) and have been using it for a few months with the on-premise version of Jira in my company.
Unfortunately, our company has switched to the cloud version of Jira and now I am having big problems connecting to the new instance.
My question: Does anyone here uses the cloud version of Jira?
What are the correct connection settings, especially the host setting? 
Is a host-config with "https://[COMPANY].atlassian.net" correct?
Currently I'm receiving the error message: "Jira: Unauthorized".
I've tried connecting with email/password combination and also with email/api-token combination. Nothing seems to work.
Thank you very much for ANY help!
Kind regards, Thomas

**Comment by Pillmayr** [comment]
SideInfo: I already tried other apps and they all work (connect) as expected (e.g. https://apps.apple.com/at/app/worklog-time-tracking/id1620141189?mt=12)

**Comment by freakkiller** [comment]
I have the same problem, but the error messages from the system are different. Unlike my colleague, I use the latest, Chrome-based version (7.13.1) after consulting various forums. Apart from the already documented error message, I also get the error messages 'Jira: Request Timed out' or 'Jira response contained invalid data', which amounts to the same thing, namely that the tool cannot currently be used properly.

The access data is correct. I downloaded the programme 'Worklog time tracking' from the AppStore and entered the same access data as for Super Productivity (also the same API token key) and the connection to our Jira was established without any problems. However, the programme only offers a minimal range of functions and is in no way comparable with Super Productivity.

**Comment by Pillmayr** [comment]
FYI: The Problem is, that we have to insert a auto-generated token of the atlassian cloud BUT without checking the setting for using the personal access token instead of the password. For me that makes absolutely no sense, but.... the main thing is that it works

**Comment by JasonFulksFamilySearch** [comment]
I also had issues... for me, closing the application (10.0.11) and then reopening solved the issue.

**Comment by jensschaerer** [comment]
I've tried connecting using the Windows App and connection works but I don't get any tasks. I changed JQL query to `assignee = currentUser()` as we don't have any sprints but no tickets show up. there is no error. I tried with another cloud instance and also there are no issues imported into Super-Productivity. Is there a bug or do I do something wrong? Link used is https://instancename.atlassian.net, Username = email-adress, password = account password.
I tried with several settings but none of the worked.

when entering the JQL in the issue search in JIRA, I get all my issues assigned to me
